estate/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+{
+    'name': 'Estate',
+    'version': '0.0',
+    'depends': ['base'],
+    'author': 'haall-odoo',
+    'application': True,
+    'installable': True,
+    'category': '',
+    'description': '',
+    'license': 'GPL-3',
```

Comment:
```suggestion
    'license': 'LGPL-3',
```
Odoo Community (Open-Source) default license is `LGPL-3`, and Enterprise is `OEEL-1`.